src/main/java/www/purple/mixxy/conf/Routes.java

```diff
@@ -80,10 +80,8 @@ public void init(final Router router) {
     // Login / Logout
     ///////////////////////////////////////////////////////////////////////////
     router.GET().route("/login/?").with(LoginLogoutController.class, "login");
-    router.POST().route("/login/?").with(LoginLogoutController.class, "loginPost");
     router.GET().route("/logout/?").with(LoginLogoutController.class, "logout");
```

Comment:
For Google and Facebook, not really since we're strictly using it to register a user.